SparkPost/2web2ui
# TR-1436: Fullscreen layout for prototype fullscreen code editor

This is a subtask [TR-1436](https://jira.int.messagesystems.com/browse/TR-1436) of [TR-1204](https://jira.int.messagesystems.com/browse/TR-1204).

### What Changed

- Replaced routing by content part for sub navigation, Contents and Settings (eventually, AB Testing)
- New layout

<img width="1391" alt="Screen Shot 2019-05-24 at 11 24 51 AM" src="https://user-images.githubusercontent.com/1335605/58338897-9541ab80-7e16-11e9-93b0-bfd77d70708f.png">


### How To Test
- Confirm routing to subnav works (e.g. /templates/edit/recipient-validation-beta-email/next/settings)
- Confirm save button still works
